sqlglot/dialects/db2.py

```diff
@@ -0,0 +1,227 @@
+
+    TIME_MAPPING = {
+        "AM": "%p",
+        "A.M.": "%p",
+        "PM": "%p",
+        "P.M.": "%p",
+        "D": "%u",
+        "DAY": "%A",
+        "DD": "%d",
+        "DDD": "%j",
+        "DY": "%a",
+        "HH": "%I",
+        "HH12": "%I",
+        "HH24": "%H",
+        "IW": "%V",
+        "MI": "%M",
+        "MM": "%m",
+        "MON": "%b",
+        "MONTH": "%B",
+        "SS": "%S",
+        "WW": "%W",
+        "YY": "%y",
+        "YYYY": "%Y",
+        "FF": "%f",
+        "FF1": "%f",
+        "FF2": "%f",
+        "FF3": "%f",
+        "FF4": "%f",
+        "FF5": "%f",
+        "FF6": "%f",
+    }
+
+    class Tokenizer(tokens.Tokenizer):
+        QUOTES = ["'", '"']
+        IDENTIFIERS = ["`"]
+
+        # Associates certain meaningful words with tokens that capture their intent
+        KEYWORDS = {
+            **Tokenizer.KEYWORDS,
+            "(+)": TokenType.JOIN_MARKER,
+            "INT64": TokenType.BIGINT,
+            "FLOAT64": TokenType.DOUBLE,
+            "CURRENT DATE": TokenType.CURRENT_DATE,
+            "CURRENT TIME": TokenType.CURRENT_TIME,
+            "CURRENT TIMESTAMP": TokenType.CURRENT_TIMESTAMP,
+            "ORDER BY": TokenType.ORDER_BY,
+        }
+
+    class Parser(parser.Parser):
+        ALTER_TABLE_ADD_REQUIRED_FOR_EACH_COLUMN = False
+        WINDOW_BEFORE_PAREN_TOKENS = {TokenType.OVER, TokenType.KEEP}
+        VALUES_FOLLOWED_BY_PAREN = False
+        ALIAS_POST_TABLESAMPLE = True
+        STRING_ALIASES = True
+
+        FUNCTIONS = {
+            **parser.Parser.FUNCTIONS,
+            "DATE_SUB": build_date_delta_with_interval(exp.DateSub),
+            "TO_CHAR": _build_timetostr_or_tochar,
+            "DATE_ADD": build_date_delta_with_interval(exp.DateAdd),
+            "DATE": lambda args: exp.TsOrDsToDate(this=seq_get(args, 0)),
+            "CHAR": lambda self: self._parse_chr(),
+            "GROUP_CONCAT": lambda self: self._parse_group_concat(),
+            # https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_values
+            "VALUES": lambda self: self.expression(
+                exp.Anonymous, this="VALUES", expressions=[self._parse_id_var()]
+            ),
+            "TO_DATE": build_formatted_time(exp.StrToDate, "db2"),
+            "DAY": lambda args: exp.Day(this=exp.TsOrDsToDate(this=seq_get(args, 0))),
+            "TO_DAYS": lambda args: exp.paren(
+                exp.DateDiff(
+                    this=exp.TsOrDsToDate(this=seq_get(args, 0)),
+                    expression=exp.TsOrDsToDate(this=exp.Literal.string("0000-01-01")),
+                    unit=exp.var("DAY"),
+                )
+                + 1
+            ),
+        }
+
+        NO_PAREN_FUNCTIONS = {
+            **parser.Parser.NO_PAREN_FUNCTIONS,
+            TokenType.CURRENT_DATE: exp.CurrentDate,
+            TokenType.CURRENT_TIME: exp.CurrentTime,
+            TokenType.CURRENT_TIMESTAMP: exp.CurrentTimestamp,
+        }
```

Comment:
These are redundant.
```suggestion
```

Reply:
Removed in 1e96557b9b8456adb61c6cc1eb879902bf2ae9d6